### DISCUSSION TITLE
The filter to show the metadata mdified by specific user doesn't work

### DISCUSSION THREAD

**Discussion by waynechungvs** [jetstreamapp/jetstream#488] [General]
When I typed my name in the specific user place, the result doesn't return my name. But I am system admin, I really did a lot of changes.

**Comment by paustint** [comment]
@waynechungvs - Does your org have a lot of users in it?

I think I found the issue, when fetching the users only the first set of users was returned and the bug was that we did not paginate through the results. And since your name presumably starts with a `w`, you were likely sorted at the end.

The fix has been deployed, can you do a hard refresh in your browser (hold shift and click refresh) and try again?

You will probably need to click the icon to re-fetch the data instead of using the cache that is in your browser.

![image](https://github.com/jetstreamapp/jetstream/assets/5461649/dc82567d-9bb6-460f-a8bb-5e02de6f0562)

**Reply by waynechungvs** [reply]
Hi @paustint surprising about your rapid reply. The original issue works now but I found a new one. When I use choose All Metadata and Specific User and Specific Time I aim to find all the metadata that modified by specific user dring specific time frame since I don't know what exactally metadata he updated so I choose All Metadata but now the return result will return all metadata in the org in the next page. For non-modified metadata will show blank in last modified which is slighly diff than what we expeted eventhough I know in the next page I still can do column level filter. But overall I think this tool is amazing.

**Reply by paustint** [reply]
@waynechungvs 
> For non-modified metadata will show blank in last modified which is slighly diff than what we expeted eventhough I know in the next page I still can do column level filter.

Was this just for certain metadata types?
All the data we have available is just what Salesforce makes available, and sometimes there is some sub-par behavior. For example, standard metadata (like custom object - Account) always shows as if it was modified in 1979 🤷 

It is possible there is a bug - so if you can provide more info about what showed up without dates, I can do some research and try to reproduce it and let you know for sure.

**Comment by waynechungvs** [comment]
So my scenario is I don't know exactly the metadata types that I've changed so I select all metadata/ specific user / modified after specific date. Then the result will return all metadata and for most of them show "No metadata found". While the expeted result is only return modified metadata and get rid of those "No metadata found".
![image](https://github.com/jetstreamapp/jetstream/assets/105181305/5fd6989f-1c8d-488e-b026-30b1fb9d949a)
